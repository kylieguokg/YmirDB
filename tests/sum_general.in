set a
set b 1 a
set c 2 b
set d 3 c
set e 4 d
list entries
sum a
sum b
sum c
sum d
sum e
pop b
list entries
sum a
sum b
sum c
sum d
sum e
push a 10
append b 2
pluck c 2
list entries
sum a
sum b
sum c
sum d
sum e
