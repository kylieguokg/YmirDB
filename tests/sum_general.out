> ok

> ok

> ok

> ok

> ok

> e [4 d]
d [3 c]
c [2 b]
b [1 a]
a []

> 0

> 1

> 3

> 6

> 10

> 1

> e [4 d]
d [3 c]
c [2 b]
b [a]
a []

> 0

> 0

> 2

> 5

> 9

> ok

> ok

> b

> e [4 d]
d [3 c]
c [2]
b [a 2]
a [10]

> 10

> 12

> 2

> 5

> 9

> bye
